its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -901,6 +960,45 @@ private void runCSharpSharedFileWithOneProjectUsingProjectBaseDir(Function<Path,
       .isNotNull();
   }
 
+  private BuildResult runAnalysisWithoutProjectBasedDir(Path projectDir)
+  {
+    String token = TestUtils.getNewToken(ORCHESTRATOR);
+    String folderName = projectDir.getFileName().toString();
+    ScannerForMSBuild scanner = TestUtils.newScanner(ORCHESTRATOR, projectDir, ScannerClassifier.NET_5)
+      .addArgument("begin")
+      .setProjectKey(folderName)
+      .setProjectName(folderName)
+      .setProjectVersion("1.0")
+      // do NOT set "sonar.projectBaseDir" for this test
+      .setProperty("sonar.login", token)
```

Comment:
ToDo: group `setProperty` together and add ` // simulate it's not on Azure Pipelines (otherwise, it will take the projectBaseDir from there)` comment here as well.